src/manifolds/Unitary.jl

```diff
@@ -0,0 +1,31 @@
+
+@doc raw"""
+    const UnitaryMatrices{n} = AbstarctUnitaryMatrices{n,ℂ,AbsoluteDeterminantOneMatrices}
+
+The manifold ``U(n)`` of ``n×n`` complex matrices such that
+
+``p^{\mathrm{H}p = \mathrm{I}_n,``
```

Comment:
This does render correctly in documentation.

Reply:
```suggestion
``p^{\mathrm{H}}p = \mathrm{I}_n,``
```

You forgot a “not” or an “in” (before correct) and I forgot a closing `}` ;)

Reply:
Yes, I meant to write "does not render correctly" :wink: